test/connection.test.js

```diff
@@ -1011,18 +1025,64 @@ test('transaction', async () => {
     {
       method: 'getSignatureStatus',
       params: [
-        '3WE5w4B7v59x6qjyC4FbG2FEKYKQfvsJwqSxNVmtMjT8TQ31hsZieDHcSgqzxiAoTL56n2w5TncjqEKjLhtF4Vk',
+        [
+          '3WE5w4B7v59x6qjyC4FbG2FEKYKQfvsJwqSxNVmtMjT8TQ31hsZieDHcSgqzxiAoTL56n2w5TncjqEKjLhtF4Vk',
+        ],
         {commitment: 'recent'},
       ],
     },
     {
       error: null,
-      result: {Ok: null},
+      result: [
+        {
+          slot: 0,
+          status: {Ok: null},
+        },
+      ],
     },
   ]);
-  await expect(connection.getSignatureStatus(signature)).resolves.toEqual({
-    Ok: null,
-  });
+
+  const response = await connection.getSignatureStatus(signature);
+  if (response !== null) {
+    expect(typeof response.slot).toEqual('number');
+    expect(response.status).toEqual({Ok: null});
+  } else {
+    expect(response).not.toBeNull();
```

Comment:
Is this pattern right?
```
if (reponse === null) {
  expect(response).not.toBeNull();
}
```

Reply:
Yeah, unfortunately needed to add the if statement to make flow happy

Reply:
Oh, flow... 😕